_posts/2022-07-26-phone-number-capture-dataset.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Investigating Label Noise in intent classification datasets and fixing it
+date: 2022-07-26
+tags: [phone-number-capture-dataset]
+categories: [Machine Learning]
+image: assets/images/demo1.jpg
+layout: post
+authors: [anirudhthatipelli]
```

Comment:
adding `adithyanarayan` as a co-author here since he worked on all the evaluations here